### ISSUE TITLE
Assist: CMD+ENTER not working

### ISSUE THREAD

**Issue by seripap** [zed-industries/community#1712]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

I am using the `Sublime Text` default keymap with some additional mappings, but I can't get cmd+enter to work on the new assistant panel. The `magic` button in the top right does work. 

```json
[
    {
        "context": "Editor",
        "bindings": {
            "ctrl-[": "editor::MovePageUp",
            "ctrl-]": "editor::MovePageDown",
            "ctrl-d": "editor::GoToDefinition",
            "ctrl-f": "editor::MoveToNextWordEnd",
            "ctrl-b": "editor::MoveToPreviousWordStart"
        }
    }
]

```

### Environment

```
Zed: v0.92.3 (stable)
OS: macOS 13.1.0
Memory: 32 GiB
Architecture: aarch64
```

### If applicable, add mockups / screenshots to help explain present your vision of the feature

<img width="624" alt="Screenshot 2023-06-28 at 17 04 25" src="https://github.com/zed-industries/community/assets/683200/bc5c0f2c-61a9-4a50-9e83-e85f136b667b">

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by ZombieHarvester** [comment]
`cmd->` also doesn't work, it's supposed to insert selected text from the last active editor pane

**Comment by JosephTLyons** [comment]
It looks like Atom, Sublime, and Textmate keymaps specifically have a binding `"cmd-enter": "editor::NewlineBelow"` that is clobbering the new internal assistant command.

As a temp fix, you might just be able to insert a block for the default assisstant commands in your custom keymap file:

```json
{
  "context": "ConversationEditor > Editor",
  "bindings": {
    "cmd-enter": "assistant::Assist",
    "cmd-s": "workspace::Save",
    "cmd->": "assistant::QuoteSelection",
    "shift-enter": "assistant::Split",
    "ctrl-r": "assistant::CycleMessageRole"
  }
}
```

The real fix will be to adjust the keymap files such that those `cmd-enter`s don't clobber the assistant.

**Comment by JosephTLyons** [comment]
This should be fixed in our next minor stable release (v0.93.1).
